spec/models/user_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
```

Comment:
- [ ] As required for the project to have tests all your models, kindly make sure you add unit tests to both `Like` and `Comment` models.

- [ ] You should also ensure that to add corresponding unit tests to all custom methods in each model such as to the `update_likes_counter` and the `update_comments_counter` methods. This is for the project requirements.

Reply:
Sure I will add these tests as well

Reply:
Thank you for the suggestions indeed these tests were missing.  I will come up with solution